src/test/resources/features/GETcomments.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Comments
+
+  Background:
+    Given I send a "POST" request to "/projects" with json body
+    """
+    {
+    "name": "Project Automation"
+    }
+    """
+    And I save the response as "Project"
+    And I send a "POST" request to "/projects/{Project.id}/stories" with json body
+    """
+    {
+    "name": "Story Automation"
+    }
+    """
+    And I save the response as "Story"
+    And I send a "POST" request to "/projects/{Project.id}/stories/{Story.id}/comments" with json body
+    """
+    {
+    "text": "Comment created by cucumber"
+    }
+    """
+    And I save the response as "Comment"
+
+  @comment
+  Scenario: GET Comments
+    When I send a GET request to "/projects/{Project.id}/stories/{Story.id}/comments"
+    Then I validate the response has status code 200

```

Comment:
You are validating only the status code.
You can validate the body response is an array 
and validate the array contains certain info, using specific preconditions.

Apply the same approach for the another endpoints